Move to metadata to `pyproject.toml` (#166)
pypa/sampleproject: README.md

[The source for this project is available here][src].

> Most of the configuration for a Python project is done in the `pyproject.toml` file,

Reviewer: ```suggestion
The metadata for a Python project is defined in the `pyproject.toml` file,
```

Reviewer: Wording it as suggested avoids the question of whether build backend configuration should/must be done in `pyproject.toml`.
